src/webui/webapplication.cpp

```diff
@@ -47,6 +47,7 @@
 #include <QUrl>
 
 #include "base/http/httperror.h"
+#include "base/global.h"
```

Comment:
It seems like unneeded.

Reply:
Yeap. Some code was removed in the master branch.
Done